File: files-app/src/androidTest/java/l/files/ui/preview/NoPreviewCacheTest.java
package l.files.ui.preview;

import static java.lang.System.currentTimeMillis;

public final class NoPreviewCacheTest
    extends PersistenceCacheTest<Boolean, NoPreviewCache> {

  @Override NoPreviewCache newCache() {
    return new NoPreviewCache(getTestContext());
  }

  @Override Boolean newValue() {
    return currentTimeMillis() % 2 == 0;
  }
}

File: ui-preview/src/androidTest/java/l/files/ui/preview/NoPreviewCacheTest.kt
package l.files.ui.preview

import java.lang.System.currentTimeMillis

internal class NoPreviewCacheTest :
  PersistenceCacheTest<Boolean, NoPreviewCache>() {

  override fun newCache() = NoPreviewCache { mockCacheDir() }
  override fun newValue() = currentTimeMillis() % 2 == 0L
}
